### PR TITLE
Pre-release updates to copyright and citations

### PR DESCRIPTION
**Pull request by hollasch**

### CODE REVIEW

**Comment by trevordblack** [comment]
We're co-authors now Steve

**Comment by hollasch** [comment]
Ha! I forgot.

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -3671,13 +3669,13 @@
   ### IEEE
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     “Ray Tracing in One Weekend.” raytracing.github.io/books/RayTracingInOneWeekend.html
-    (accessed MMM. DD, YYYY)
+    (accessed MMM. DD, 2023)
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:3672]: I'm not so sure the `accessed` tag should be changed.

Aren't you supposed to clarify this as the reader when you make a citation?

**Review by trevordblack — APPROVED** [review]
Looks good other than the `accessed` tag in citations

File: books/RayTracingInOneWeekend.html (hunk 1 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html:3672]: Will fix.

**Pull request merged**